.github/workflows/deploy_stage.yaml

```
@@ -14,12 +14,15 @@ on:
 
 env:
   REGISTRY: ghcr.io
+  DEPLOY_PATH: /RandomCoffeeBot
```

Предлагаю использовать домашнюю папку 
```suggestion
  DEPLOY_PATH: ~/app  # Точно не знаю, будет ли работать `~`. Если нет, используй `/home/devMattermost/app`
```